server/interpreter.js

```diff
@@ -390,6 +371,9 @@ Interpreter.prototype.initObject = function(scope) {
   };
 
   // Static methods on Object.
+  this.setProperty(ObjectConst, 'is', this.createNativeFunction(Object.is),
```

Comment:
As is `Object.setPrototypeOf()`…  And IIRC there was at least one feature we need that turned out to be ES7!